cmake/SpectreParseTests.py

```diff
@@ -6,7 +6,8 @@
 import re
 
 # A list of all the allowed ctest labels/Catch tags for tests
-allowed_tags = ["CompilationTest",
+allowed_tags = ["ApparentHorizons",
```

Comment:
Should we start the list on a new line to avoid conflicts like this one? (Just as we do on the last line)